packages/spotlight/src/container.js

```diff
@@ -328,7 +364,7 @@ const getSpottableDescendants = (containerId) => {
 
 	candidates.push(...getOwnedNodes(node, selector));
 
-	return candidates.filter(n => navigableFilter(n, containerId));
+	return candidates.filter(n => isNavigable(n, containerId));
```

Comment:
We need a discussion about function names, especially between the terms 'Spottable' and 'Navigable'.

Reply:
We're going to change `navigableFilter` rather than `getSpottableDescendants`.